### PR TITLE
Handle edge case when node might be null when emitting 'generated from' comment.

### PR DESCRIPTION
**Pull request by nbeloglazov**
Tested:
  there is no test for that edge case as I couldn't force it on clean unit test
  tested building some internal Google projects

### CODE REVIEW

File: src/main/java/com/google/javascript/clutz/DeclarationGenerator.java [hunk 1]
```diff
@@ -3350,26 +3350,21 @@ void walkInnerSymbols(ObjectType type, String innerNamespace) {
         String qualifiedName = innerNamespace + '.' + propName;
         if (provides.contains(qualifiedName)) continue;
         Node node = innerProps.get(namedType);
+        // Start namespace if this is the first pair being processed.
+        if (!foundNamespaceMembers &&
+            (pType.isEnumType() || isClassLike(pType) || isTypedef(pType))) {
```
Comment (rkirov) [inline src/main/java/com/google/javascript/clutz/DeclarationGenerator.java]: This is fragile because the clauses here have to match the if stmts below. At least add a comment to that effect.

Comment (nbeloglazov) [reply src/main/java/com/google/javascript/clutz/DeclarationGenerator.java]: Changed it back. Couldn't think of nice way to reuse the code and prevent possible issue that you mentioned. So decided not worth it :)

**Comment by nbeloglazov** [comment]
Done. I've no idea how to do it properly so I just created new branch from master and applied my changes there. One I need to properly learn git rebase.
